src/mb_enhanced_cover_art_uploads/providers/vgmdb.ts

```diff
@@ -146,6 +147,47 @@ export class VGMdbProvider extends CoverArtProvider {
     urlRegex = /\/album\/(\d+)(?:\/|$)/;
 
     async findImages(url: URL): Promise<CoverArt[]> {
+        const pageSrc = await this.fetchPage(url);
+        if (pageSrc.includes('/db/img/banner-error.gif')) {
+            throw new Error('VGMdb returned an error');
+        }
+
+        const pageDom = parseDOM(pageSrc, url.href);
+
+        // istanbul ignore else: Tests are not logged in
+        if (qsMaybe('#navmember', pageDom) === null) {
+            LOGGER.warn('Heads up! VGMdb requires you to be logged in to view all images. Some images may have been missed. If you have an account, please log in to VGMdb to fetch all images.');
+        }
+
+        const coverGallery = qsMaybe('#cover_gallery', pageDom);
+        const galleryCovers = coverGallery ? await VGMdbProvider.extractCoversFromDOMGallery(coverGallery) : [];
+
+        // Add the main cover if it's not in the gallery
+        const mainCoverUrl = qsMaybe<HTMLDivElement>('#coverart', pageDom)?.style.backgroundImage.match(/url\(["']*(.+?)["']*\)/)?.[1];
```

Comment:
```suggestion
        const mainCoverUrl = qsMaybe<HTMLDivElement>('#coverart', pageDom)?.style.backgroundImage.match(/url\(["']?(.+?)["']?\)/)?.[1];
```
Is it possible that the URL is wrapped in multiple quotes?

Reply:
Probably not 🙂 In JSDom, they're not wrapped in quotes at all, but in FF it appears they are.